docs/tutorials/create_expectations.rst

```diff
@@ -3,264 +3,214 @@
+:ref:`Data Docs<data_docs>` is a feature of Great Expectations that creates data documentation by compiling expectations and validation results into HTML.
 
+Data Docs produces a visual data quality report of what you expect from your data, and how the observed properties of your data differ from your expectations.
+It helps to keep your entire team on the same page as data evolves.
 
-.. content-tabs::
+Reviewing expectations is best done in Data Docs:
 
-    .. tab-container:: tab0
-        :title: pandas
+.. image:: ../images/sample_e_s_view.png
 
-        A Pandas datasource generates data assets from Pandas DataFrames or CSV files. In this example the pipeline processes NPI data that it reads from CSV files in the ``npidata`` directory into Pandas DataFrames. This is the data you want to describe with expectations. That directory and its files form a data asset, named "npidata" (based on the directory name).
+Edit an Expectation Suite
+----------------------------------------
 
-    .. tab-container:: tab1
-        :title: pyspark
+The best interface for editing an Expectation Suite is a Jupyter notebook.
 
-        A Spark datasource generates data assets from Spark DataFrames or CSV files. The data loaded into a data asset is the data you want to describe and specify with expectations. If this example read CSV files in a directory called ``npidata`` into a Spark DataFrame, the resulting data asset would be called "npidata" based on the directory name.
+Editing an Expectation Suite means adding expectations, removing expectations, and modifying the arguments of existing expectations.
 
-    .. tab-container:: tab2
-        :title: SQLAlchemy
+For every expectation type there is a Python method that sets its arguments, evaluates this expectation against a sample batch of data and adds it to the Expectation Suite.
 
-        A SQLAlchemy datasource generates data assets from tables, views and query results.
+Take a look at the screenshot below. It shows the HTML view and the Python method for the same expectation (``expect_column_distinct_values_to_be_in_set``) side by side:
 
-        * If the data resided in a table (or view) in a database, it would be accessible as a data asset with the name of that table (or view).
-        * If the data did not reside in one table ``npidata`` and, instead, the example pipeline ran an SQL query that fetched the data (probably from multiple tables), the result set of that query would be accessible as a data asset. The name of this data asset would be up to us (e.g., "npidata" or "npidata_query").
+.. image:: ../images/exp_html_python_side_by_side .png
 
+The CLI provides a command that, given an Expectation Suite, generates a Jupyter notebook to edit it. It takes care of generating a cell for every expectation in the suite and of getting a sample batch of data. The HTML page for each Expectation Suite has the CLI command syntax in order to make it easier for users.
```

Comment:
No change suggested...but this is pretty cool and should have its own feature page.